### ISSUE TITLE
Using PyInstaller

### ISSUE THREAD

**Issue by DarkLord76865** [TomSchimansky/TkinterMapView#72]
Hi,
does this work with PyInstaller --onefile ? I've seen that it installs CustomTkinter as dependency, but on repo for CustomTkinter it is said that it does not work with --onefile. I don't need any CustomTkinter functionality, just want this TkinterMapView widget, but I'm using --onefile. 
Also, is it possible to set like map edges? I don't want to let user move map to the Antarctica, just want to let the map be moved/zoomed within certain area.
Thanks

**Comment by twintersx** [comment]
@DarkLord76865 would also like to use PyInstaller. Have you found a solution?

**Comment by DarkLord76865** [comment]
> @DarkLord76865 would also like to use PyInstaller. Have you found a solution? 

I unfortunately don't have time to check if it works with PyInstaller or not. If it doesn't, I will just try to switch to Pyside6 instead of tkinter. It will be a learning curve, but it is probably more future proof than tkinter. (I have seen that there are map objects for qt)

**Comment by DarkLord76865** [comment]
> @DarkLord76865 would also like to use PyInstaller. Have you found a solution? 

I've tried it out, it mostly works correctly with PyInstaller --onefile. That said the only thing that doesn't work is the only thing that is dependent on custom tkinter, right click menu onto map. In images provided in README.md, right click menu looks super cool, I suppose that it depends on custom tkinter. When I tested I didn't get that fancy right click menu, I just got plain old white windows style menu. Though, I don't mind and will use this library for my project. I even added functionality for using local overlay tiles and put pull request for it, hope it gets accepted.
